.php_cs.dist

```diff
@@ -39,5 +39,8 @@ return \PhpCsFixer\Config::create()
     ->setFinder(
         \PhpCsFixer\Finder::create()
             ->in(getcwd())
+            ->exclude('var/cache')
+            ->notPath('var/SymfonyRequirements.php')
```

Comment:
These files should be removed from your Symfony project and the composer scripts regenerating them should also be removed (`"Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile`). See https://github.com/sensiolabs/SensioDistributionBundle/blob/master/Composer/ScriptHandler.php#L183